packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,7 +101,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root'>
         { label && (
-          <label id={ id + '-label' } className='ms-Label' ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } >{ label }</label>
+          <Label id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
<Label htmlFor={ id }>{ label }</Label>

I think "htmlFor" maybe a better answer than using "id".

Further not sure why you are using ref= _dropdownLabel.  I don't see it being used anywhere.